utilitary-itest/pom.xml

```
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>elementary-project</artifactId>
+        <groupId>com.karuslabs</groupId>
+        <version>1.2.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>utilitary-itest</artifactId>
+    <packaging>jar</packaging>
+    <name>Utilitary integration test</name>
```

@Pante you can also add suggestions like so:
```suggestion
    <name>Utilitary Integration Tests</name>
```

---

Ohhh, that’s really neat, I never knew that feature existed.